src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI
+{
+    namespace WindowSystem
+    {
+        namespace WsShell
+        {
+            /// <summary>
+            /// Class for the Tizen quickpanel client.
+            /// </summary>
+            public class QuickPanelClient : IDisposable
+            {
+                private TzShell _tzsh;
+                private IntPtr _tzshQpClient;
+                private int _tzshWin;
+                private bool disposed = false;
+
+                private int _visibleEventType;
+                private IntPtr _visibleEventHandler;
+                private Interop.QuickPanelClient.QuickPanelEventCallback _OnVisibleChanged;
+                private event EventHandler<StateVisible> _VisibleEventHandler;
+                /// <summary>
+                /// Emits the event when the visible is changed.
+                /// </summary>
+                /// <exception cref="ArgumentException">This exception can be due to an invalid state.</exception>
+                public event EventHandler<StateVisible> VisibleEventHandler
```

Comment:
Rename event handler : https://docs.microsoft.com/ko-kr/dotnet/standard/design-guidelines/names-of-type-members#names-of-events

Reply:
I changed VisibleEventHandler -> VisibleChanged